log.go

```diff
@@ -48,6 +50,7 @@ var subsystemLoggers = map[string]btclog.Logger{
 	"UTXN": utxnLog,
 	"BRAR": brarLog,
 	"CMGR": cmgrLog,
+	"PROP": htswLog,
```

Comment:
Also, the htlcSwitch already has its own sub-system logger, the creation of another one is not necessary.